gulpfile.js

```diff
@@ -24,7 +24,7 @@ gulp.task('browser-sync', function() {
       baseDir: "./build/"
     },
     files: ['./build/**/*.*'],
-    browser: 'google chrome',
+    browser: 'chrome',
```

Comment:
let's go ahead and try with out any {{browser:}} entry in here.  We had a similar issue [in MapTable](https://github.com/Packet-Clearing-House/maptable/commit/b21cca9b763df64b8500e919867b14aac59d8738) and removing this line solved it. 

Give it a try and let report back!